recipes/recipe-1.html

```diff
@@ -0,0 +1,64 @@
+<html lang="en"><link type="text/css" rel="stylesheet" id="dark-mode-custom-link"><link type="text/css" rel="stylesheet" id="dark-mode-general-link"><style lang="en" type="text/css" id="dark-mode-custom-style"></style><style lang="en" type="text/css" id="dark-mode-native-style"></style><style lang="en" type="text/css" id="dark-mode-native-sheet"></style><head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Connor's Bakery</title>
+    <link rel="stylesheet" href="/index.css">
```

Comment:
For your relative paths remember that if a file in is in a directory / folder that is above the current one you need to add `..` before the `/`. If it is in the current folder remember to add `.` before the `/`. For example this reference to index.css would be `../index.css` so that your styling loads!